### PR TITLE
docs: watermark

### PR DESCRIPTION
**Pull request by LiuJi-Jim**
### Structure

- Administration
  - ...
  - Database Access Control
  - Audit Log
  - Anonymize Data
  - **Watermark**
  - Webhook Integration
  - ...

### Screenshots

image

### CODE REVIEW

File: content/docs/administration/watermark.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+title: Watermark
+---
+
+<hint-block type="info">
+
+This feature is only available in the Enterprise plan.
+
+</hint-block>
+
+Workspace Owners and DBAs can turn on / off watermark setting in **"Settings/ Workspace / General"**
```
Comment (tianzhou) [inline content/docs/administration/watermark.md:11]: Synced with @Candybase offline. Let's only allow workspace Owner to change watermark.

The reason is watermark is a general feature (non-db tool will also have a watermark feature), thus it's less useful to grant it to DBA.

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/administration/watermark.md (hunk 1 above)
Comment (LiuJi-Jim) [reply content/docs/administration/watermark.md:11]: sry. The code implementation is Owner only, the doc is wrong. Updating in progress.

Comment (LiuJi-Jim) [reply content/docs/administration/watermark.md:11]: Updated.

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/administration/watermark.md (hunk 1 above)
Comment (LiuJi-Jim) [reply content/docs/administration/watermark.md]: P.S. we used the same privilege identifier as the Branding setting ("bb.permission.workspace.manage-general")

**Pull request merged**